packages/react/.babelrc

```diff
@@ -1,6 +1,6 @@
 {
   "plugins": ["annotate-pure-calls"],
-  "presets": ["@babel/preset-react"],
+  "presets": ["@babel/preset-react", "@babel/typescript"],
```

Comment:
Added to support typescript in storybook.
Probably not the correct way, will continue looking into this.

Reply:
Should I just use `@nrwl/react/babel` as in `test` for `development` or always?

Reply:
Moved to storybook config.